### PR TITLE
Paul/moonpay sell2

### PR DESCRIPTION
**Pull request by paullinator**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208206905835937

### CODE REVIEW

File: src/plugins/gui/providers/moonpayProvider.ts [hunk 1]
```diff
@@ -356,27 +384,194 @@ export const moonpayProvider: FiatProviderFactory = {
...
+
+                    if (uri.startsWith(RETURN_URL_PAYMENT)) {
+                      console.log('Moonpay WebView launch payment: ' + uri)
+                      const urlObj = new URL(uri, true)
+                      const { query } = urlObj
+                      const { baseCurrencyAmount, baseCurrencyCode, depositWalletAddress, depositWalletAddressTag, transactionId } = query
+                      if (inPayment) return
+                      inPayment = true
+                      try {
+                        if (baseCurrencyAmount == null || baseCurrencyCode == null || depositWalletAddress == null || transactionId == null) {
```
Comment (samholmes) [inline src/plugins/gui/providers/moonpayProvider.ts:448]: Are we in control over these query parameters? Or is this something we expect moonpay to provide during the URL change event?

File: src/plugins/gui/providers/paybisProvider.ts [hunk 2]
```diff
@@ -64,6 +65,7 @@ const asApiKeys = asObject({
 const asPaymentMethodId = asValue(
   'method-id-credit-card',
   'method-id-credit-card-out',
+  'method-id_bridgerpay_revolutpay',
```
Comment (samholmes) [inline src/plugins/gui/providers/paybisProvider.ts:68]: What's the rule for `-` vs `_`?

**Review by samholmes — APPROVED** [review]
Approved but just some questions while reviewing.

File: src/plugins/gui/providers/paybisProvider.ts (hunk 2 above)
Comment (paullinator) [reply src/plugins/gui/providers/paybisProvider.ts:68]: I don't know. It's Paybis and what they want to call these values. We have no control.

File: src/plugins/gui/providers/moonpayProvider.ts (hunk 1 above)
Comment (paullinator) [reply src/plugins/gui/providers/moonpayProvider.ts:448]: moonpay provides these values. we have no control.

**Pull request merged**